index.html

```diff
@@ -2001,6 +1924,50 @@ <h3>Verification Method types</h3>
         </pre>
 
       </section>
+
+      <section>
+        <h3>Referring to Verification Methods</h3>
+        <p>
+<a>Verification methods</a> can be embedded in or referenced from properties
+associated with various <a>verification relationships</a> as described in <a
+href="#verification-relationships"></a>. Referencing <a>verification methods</a>
+allows them to be used by more than one <a>verification relationship</a>.
+        </p>
+
+        <p>
+If the value of a <a>verification method</a> property is a <a
+data-cite="INFRA#ordered-map">map</a>, the <a>verification method</a> has been
+embedded and its properties can be accessed directly. However, if the value is a
+URL <a data-cite="INFRA#string">string</a>, the <a>verification method</a> has
+been included by reference and its properties will need to be retrieved from
+elsewhere in the <a>DID document</a> or from another <a>DID document</a>. This
```

Comment:
is "from another DID document" accurate, or is that a security vulnerability? (Removing this line would be a non-normative change, I think).

Reply:
Whether or not it's a security vulnerability depends on the system consuming it. It's certainly a security consideration.